scene/2d/tile_map.cpp

```diff
@@ -1151,6 +1151,8 @@ void TileMap::_navigation_update_layer(int p_layer) {
 			layers[p_layer].uses_world_navigation_map = true;
 		} else {
 			RID new_layer_map = NavigationServer2D::get_singleton()->map_create();
+			// Set the default NavigationPolygon cell_size on the new map as a mismatch causes an error.
+			NavigationServer2D::get_singleton()->map_set_cell_size(new_layer_map, 1.0);
```

Comment:
Wouldn't it be possible to extract this constant (`1.0`) to some shared place so that it's not defined in multiple places?

Reply:
Possible yes but outside the scope of this pr. Changing this to shared constants for the defaults would require more changes and header includes all over the place in the engine.

Reply:
Ok, added to https://github.com/godotengine/godot/issues/73566 tracker